handbook.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: contribute
+title: Handbook
+---
+
+# How we make things
+
+## Branching and SCM
+
+We use git for source control. Our projects in
+[theforeman](https://github.com/theforeman) follow this model for branching:
+
+* **develop**
+    * All pull requests are merged here.
```

Comment:
Perhaps expand this to say that all PRs should be made against this branch, and are merged here?